setup.sh

```diff
@@ -78,13 +86,13 @@ function logTimestamp() {
 }
 
 function setupTimezone() {
-    echo -ne "Enter the timezone for the server (Default is 'Asia/Singapore'):\n" >&3
+    echo -ne "Enter the timezone for the server (Default is 'Europe/Rome'):\n" >&3
     read -r timezone
     if [ -z "${timezone}" ]; then
-        timezone="Asia/Singapore"
+        timezone="Europe/Rome"
```

Comment:
Readme documentation refers to "Asia/Singapore". 
Could you advise why this should be changed, and also update the readme. 
Otherwise, please change the line back to "Asia/Singapore"

Reply:
I simply use in all my vps europe/rome

I added few lines to the readme plus some info on ssh derivation